model/event/subscriptions.go

```diff
@@ -323,6 +323,11 @@ func (s *Subscription) Validate() error {
 	if !IsValidOwnerType(string(s.OwnerType)) {
 		catcher.Add(errors.Errorf("%s is not a valid owner type", s.OwnerType))
 	}
+	if s.OwnerType == OwnerTypeProject &&
```

Comment:
Can you add a comment about why we're validating this?

Reply:
done